### PR TITLE
Test WorkflowSpec.match()

### PR DESCRIPTION
**Pull request by jwodder**

### CODE REVIEW

File: src/tinuous/github.py [hunk 1]
```diff
@@ -64,7 +63,7 @@ def ghrepo(self) -> Repository:
 
     def get_workflows(self) -> Iterator[Workflow]:
         for wf in self.ghrepo.get_workflows():
-            if self.workflow_spec.match(removeprefix(wf.path, ".github/workflows/")):
+            if self.workflow_spec.match(wf.path):
```
Comment (yarikoptic) [inline src/tinuous/github.py:66]: what is the motivation for this change/PR?  I think I have used/relied already on IIRC documented ability to just specify workflow name without leading to it paths, and test below suggests that it would now require full relative name, or am I mistaken and taking `.name` above would still be in effect and user would not really need to specify full path?

Comment (jwodder) [reply src/tinuous/github.py:66]: This just moves the stripping of the `.github/workflows/` to `match()`.  This was partially motivated by the fact that, when I originally wrote the code, I accidentally swapped the order of the `removeprefix()` arguments, and it took a while to debug.  Now, the basenaming of the workflow path is covered by the tests.

Comment (yarikoptic) [reply src/tinuous/github.py:66]: great, thank you for the explanation! IMHO it was worth placing into a commit message (and propagate into PR description).
